common/workunit/workunit.cpp

```diff
@@ -13595,6 +13595,8 @@ void executeThorGraph(const char * graphName, IConstWorkUnit &workunit, const IP
 {
     unsigned timelimit = workunit.getDebugValueInt("thorConnectTimeout", config.getPropInt("@thorConnectTimeout", 60));
     StringAttr wuid(workunit.queryWuid());
+    IConstWUGraph *graph = workunit.getGraph(graphName);
+    unsigned wfid = graph ? graph->getWfid() : 0;
```

Comment:
is graph ever expected to be null? if so how?
If it's not expected, then it's going to go wrong soon after this, would be better to catch here and throw a meaningful error.